README.rst

```diff
@@ -102,3 +102,51 @@ Perform a query
 
     for row in rows:
         print(row.name)
+
+Instrumenting With OpenTelemetry
+--------------------------------
+
+This application uses OpenTelmetry to output tracing data from
+api calls to BigQuery. To enable OpenTelemetry tracing in
+the BigQuery client the following PyPi packages need to be installed:
+
+.. code-block:: console
+
+    pip install opentelemetry-api
```

Comment:
Generally, we define these as optional dependencies using `extras`. See `setup.py` for examples. Then, these instructions become `pip install google-cloud-bigquery[opentelemetry]`